File: SoloLearn Codes/Jungle Camping/JungleCamping.java
import java.util.*;
import java.util.stream.*;

public class JungleCamping
{
    public static void main(String[] args) {
        String[] animals = new Scanner(System.in).nextLine().split(" ");
        Map<String, String> animalList = Map.of("Grr", "Lion", "Rawr", "Tiger", "Ssss", "Snake", "Chirp", "Bird");
		System.out.println(Arrays.stream(animals)
                .filter(a -> animalList.containsKey(a))
                .map(a -> animalList.get(a))
                .collect(Collectors.joining(" ")));
	}
}

File: SoloLearn Codes/Jungle Camping/jungle_camping.c
#include <stdio.h>
#include <string.h>
 
int main() {
    char str[100];    
    fgets(str, 100, stdin);    
    char *token = strtok(str, " ");  
    while(token) {
        if(strcmp(token, "Grr") == 0) puts("Lion ");
        if(strcmp(token, "Rawr") == 0) puts("Tiger ");
        if(strcmp(token, "Ssss") == 0) puts("Snake ");
        if(strcmp(token, "Chirp") == 0) puts("Bird ");
        token = strtok(NULL, " ");
    }    
    return 0;
}